formula/src/main/java/com/instacart/formula/Stream.kt

```diff
@@ -20,11 +20,11 @@ interface Stream<Message> {
     companion object {
 
         /**
-         * Emits a message when [Stream] is initialized. Use this stream to send a message when [Formula]
+         * Emits a effects when [Stream] is initialized. Use this stream to send a effects when [Formula]
```

Comment:
`Emits effects` (remove the a). `Use this stream to send effects` (remove the a)